deploy/kubernetes/filebeat-autodiscover-k8s.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
```

Comment:
Could you share a bit of background on the addition of this file? Does not seem to be mentioned in the description. I can see how it's related to the coredns module but not sure if we should address / add it in a separate PR? @jsoriano Thougts?

Reply:
If there is something that could be added to the files we already have, and is related to this PR, I'd add it here, but to the files we already have, not as a new one.

If we want to offer a different new way to deploy filebeat, yes, please open another PR for that.

Reply:
The current filebeat file under deploy/kubernetes does not meet the configuration requirement of coredns, hence this file was added because I did not want to make changes to the existing one. Thanks for the feedback and I will give that a try instead.

Reply:
I am going to remove the file and put all the details in the README instead.